site/src/components/Playground.scss

```diff
@@ -118,6 +118,10 @@
   }
 }
 
+.clipped-view-area {
```

Comment:
Just a nuisance but could/should this class be named something like .playground-clipped-view-area etc?

Reply:
Else looks good though 👍

Reply:
The class name is in Playground.scss and used only in Playground, so adding that seems overstating the fact it is for playground. 😄 

But all other classes have that, so I'll add it. Good catch!

Reply:
Fixed.